pkg/types/types.go

```diff
@@ -16,9 +16,12 @@ type NetConf struct {
 	HostIFNames string // VF netdevice name(s)
 	HostIFGUID  string // VF netdevice GUID
 	ContIFNames string // VF names after in the container; used during deletion
-	GUID        string `json:"guid"`
+	GUID        string // VF Guid is allowed only read from cni-args of network attachment
```

Comment:
Why guid define here?

Reply:
The guids has a json tag `"-"` so it won't be read when parse the configurations.
This is the config that will be passed to the functions (ApplyVFConfig and ResetVFConfig), do you prefer it as a separate argument?